ga4gh/datamodel/ontologies.py

```diff
@@ -27,10 +27,20 @@ def add(self, id_, name):
         self._idNameMap[name] = id_
 
     def getId(self, name):
+        try:
```

Comment:
This seems problematic. Usually failing noisily is a good thing. We should provide methods like ``hasId`` and ``hasName`` for the cases when we want to know if a given id/name exists in the map.